CHANGELOG.md

```diff
@@ -29,13 +29,13 @@ All notable changes to this project will be documented in this file.
 - opa: Add version `0.61.0` ([#538]).
 - spark: Add version `3.4.2` ([#560]).
 - superset: Add version `2.1.3`,`3.0.3`,`3.1.0` ([#563]).
-- trino: Add version `440` ([#587]).
+- trino: Add version `442` ([#597]).
```

Comment:
I have been leaving in the previous PR as well. Not sure if we are supposed to, but in case you want to...

```suggestion
- trino: Add version `442` ([#587], [#597]).
```